bulk_sync_into_benchling/bulk_sync_into_benchling.py

```diff
@@ -0,0 +1,205 @@
+import json
+
+import click
+import requests
+import time
+
+
+class BadRequestException(Exception):
+    def __init__(self, message, rv):
+        super(BadRequestException, self).__init__(message)
+        self.rv = rv
+
+
+def api_get(domain, api_key, path):
+    url = "https://{}/api/v2/{}".format(domain, path)
+    rv = requests.get(url, auth=(api_key, ""))
```

Comment:
Generally I'd recommend that clients use a `Session`. Making requests this way means that the SSL and the TCP connection must be re-established for every request. Up to you if you think that makes the example too complex though.

Reply:
We should definitely make that example but I think its out of scope for this PR.

Reply:
I would also like an example of polling a listing endpoint.